packages/docs/src/styles/index.scss

```diff
@@ -25,3 +25,11 @@
 @import 'components/tableOfContents.scss';
 @import 'components/spacingUtilityExampleList.scss';
 @import 'components/video.scss';
+
+[data-theme='healthcare'] {
+  @import '@cmsgov/ds-healthcare-gov/src/styles/index';
```

Comment:
Previously it only imported the components index, which was [the equivalent of importing the seven healthcare style modules](https://github.com/CMSgov/design-system/blob/main/packages/ds-healthcare-gov/src/styles/index.scss#L13-L19). Right now this is including the whole set of DS styles again under the namespace. If we can be more surgical here, we can save on a lot of bytes.

Reply:
You're right, this was a lazy addition, updating